src/components/Class/ClassCard/index.js

```diff
@@ -176,72 +181,101 @@ function ClassCard({ item, editClass, enroll, style }) {
     });
   };
   return (
-    <div className="class-card ">
-      <div className="class-details">
-        <span className="class-type">
-          {languageMap[item.type]}
-          {item.enrolled == true ? (
-            <i className="check-icon check-icon fa fa-check-circle">
-              {" "}
-              Enrolled
-            </i>
-          ) : null}
-        </span>
-        <h4>{item.title}</h4>
-        <p>Facilitator : {item.facilitator.name} </p>
-        <p>Language : {languageMap[item.lang]} </p>
-        <p>Date:{moment(classStartTime).format("DD-MM-YYYY")} </p>
-        <p>
-          Time:{moment(classStartTime).format("hh:mm a")} -{" "}
-          {moment(classEndTime).format("hh:mm a")}
-        </p>
-        <div className="bottom-details">
-          {!item.enrolled ? (
-            loading ? (
+    <>
+      {" "}
+      <Card
+        elevation={2}
+        // sx={{ p: 3, background: "#e9f5e9", width: "320px" }}
+        className={classes.card}
+      >
+        <div>
+          <Typography
+            variant="subtitle1"
+            color="primary"
+            className={classes.spacing}
+          >
+            {languageMap[item.type]}
+            {item.enrolled == true ? (
+              <i className="check-icon check-icon fa fa-check-circle">
+                {" "}
+                Enrolled
+              </i>
+            ) : null}
+          </Typography>
+          <Typography variant="subtitle1" className={classes.spacing}>
+            {item.title}
+          </Typography>
+        </div>
+        <div>
+          <Typography className={classes.spacing}>
+            Facilitator : {item.facilitator.name}
+          </Typography>
+          <Typography className={classes.spacing}>
+            Date:{moment(classStartTime).format("DD-MM-YYYY")}{" "}
```

Comment:
Since you're migrating away from Moment.js to date-fns, I'd recommend exporting a function from utils.js and importing it here:
```
const format = (d, formatStr) => moment(d).format(formatStr);
```
Then you'd use the following instead.
```suggestion
            Date:{format(classStartTime, "DD-MM-YYYY")}
```

The advantage of doing it this way is that when you migrate to `date-fns`, you could change the function in one place to use date-fns or change it again in the future if you decide to use a new date utility library without opening all the files where you're using it.